Source/mainmenu.h

```diff
@@ -20,6 +20,7 @@ BOOL mainmenu_single_player();
 BOOL mainmenu_init_menu(int type);
 BOOL mainmenu_multi_player();
 void mainmenu_play_intro();
+void mainmenu_restart_repintro();

```

Comment:
This should live in SourceX/DiabloUI/diabloui.h

Reply:
Of course, that makes more sense.
I added it to the end of diabloui.h, with a newline separating it from the rest.